appservice/src/main/resources/static/swagger/swagger-spec.yaml

```diff
@@ -2372,6 +2373,26 @@ definitions:
     required:
       - dropBuddyReason
       - _links
+  BuddyInfoChangeMessage:
```

Comment:
Going through the code a bit further, I understand the intention of this name: the message indicates a change in the buddy object, which only carries the nickname, a status and a reference to the user.

Still, I stick with my proposal: let's have a single message for changes in user information. For now, it only comes for a nickname change. Later, it'll also come for changes in the name or mobile number. When we add the photo, we'll also use this, but that'll again be a change in the buddy object.